dartagnan/src/main/java/com/dat3m/dartagnan/wmm/Wmm.java

```diff
@@ -182,10 +161,10 @@ public BooleanFormula encode(SolverContext ctx) {
         return enc;
     }
 
-    // Encodes all axioms. This should be called after <encode>
-    public BooleanFormula consistent(SolverContext ctx) {
+    // Encodes all axioms. This should be called after <encodeRelations>
```

Comment:
Add a boolean field to the class and set it to true when `encodeRelations` is called.
Then we can assert that variable when this method is called

Reply:
I think this code is already present there. I guess you only read the comment? :)